controllers/module_reconciler.go

```diff
@@ -131,7 +134,7 @@ func (r *ModuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		}
 		if requeue {
 			logger.Info("Build requires a requeue; skipping handling driver container for now", "kernelVersion", kernelVersion, "image", m)
-			res.Requeue = true
+			res.RequeueAfter = time.Second * buildInProgressTimeout
```

Comment:
Look like this is the interval waiting time. Should we change the const name to `buildInProgressInterval`?

Reply:
what about buildInProgressRequeueInterva ?l

Reply:
Sure. Just the `Timeout` word here is confusing - I get the same confusion in the commit message as well BTW.

Reply:
fixed. Corrected the commit message also

Reply:
TY